core/whistle_config-1.0.0/src/whapps_config.erl

```diff
@@ -401,10 +401,20 @@ update_category(Category, JObj, PvtFields) ->
 -spec maybe_save_category(ne_binary(), wh_json:object(), api_object(), boolean()) ->
                                  {'ok', wh_json:object()} |
                                  {'error', 'conflict'}.
+-spec maybe_save_category(ne_binary(), wh_json:object(), api_object(), boolean(), any()) ->
+                                 {'ok', wh_json:object()} |
+                                 {'error', 'conflict'}.
 maybe_save_category(Category, JObj, PvtFields) ->
     maybe_save_category(Category, JObj, PvtFields, 'false').
 
 maybe_save_category(Category, JObj, PvtFields, Looped) ->
+    IsLocked =  wh_config:get_atom('whistle_apps', 'lock_whapps_config', ['false']),
+    maybe_save_category(Category, Keys, Value, Node, Options, IsLocked).
```

Comment:
i believe you want this `maybe_save_category(Category, JObj, PvtFields, Looped, IsLocked)`